src/main/java/domain/Line.java

```diff
@@ -6,6 +6,7 @@
 
 public class Line {
 
+    public static final int ONE_STEP = 1;
     private final List<StepPoint> stepPoints = new ArrayList<>();
```

Comment:
초기화가 두 곳으로 분리되어 있는데, 생성자 쪽에 몰려있으면 초기화 로직을 한눈에 파악할 수 있을거 같습니다.
```suggestion
    private final List<StepPoint> stepPoints;

    public Line(int numberOfCell, StepGenerator stepGenerator) {
        List<StepPoint> stepPoints = new ArrayList<>();
        stepPoints.add(stepGenerator.generate());
        for (int cellIndex = 1; cellIndex < numberOfCell; cellIndex++) {
            stepPoints.add(makeOnePoint(cellIndex, stepGenerator));
        }
        
        this.stepPoints = stepPoints;
    }

```